### PR TITLE
Update for audio from speaker

### PR DESCRIPTION
**Pull request by dhalloop7**
Update for audio from the speaker instead of the earpiece for Android and iOS.

### CODE REVIEW

File: android/src/main/kotlin/vn/casperpas/sound_stream/SoundStreamPlugin.kt [hunk 1]
```diff
@@ -317,7 +317,16 @@ public class SoundStreamPlugin : FlutterPlugin,
                 .setUsage(AudioAttributes.USAGE_MEDIA)
                 .setFlags(AudioAttributes.FLAG_AUDIBILITY_ENFORCED)
                 .build()
-        mAudioTrack = AudioTrack(audioAttributes, mPlayerFormat, mPlayerBufferSize, AudioTrack.MODE_STREAM, AudioManager.AUDIO_SESSION_ID_GENERATE)
+//        mAudioTrack = AudioTrack(audioAttributes, mPlayerFormat, mPlayerBufferSize, AudioTrack.MODE_STREAM, AudioManager.AUDIO_SESSION_ID_GENERATE)
+
+        mAudioTrack = AudioTrack(AudioManager.STREAM_MUSIC,
```
Comment (CasperPas) [inline android/src/main/kotlin/vn/casperpas/sound_stream/SoundStreamPlugin.kt:322]: This AudioTrack construction is deprecated.

**Comment by CasperPas** [comment]
Thanks for your PR! I'm testing it. I'm so freaking busy these days.

File: android/src/main/kotlin/vn/casperpas/sound_stream/SoundStreamPlugin.kt (hunk 1 above)
Comment (dhalloop7) [reply android/src/main/kotlin/vn/casperpas/sound_stream/SoundStreamPlugin.kt:322]: Ohh. Did you found any other alternative for the same?

**Comment by CasperPas** [comment]
This was transfered/implemented in the following PR. I'll close this one. Thanks!
https://github.com/CasperPas/flutter-sound-stream/pull/11

**Pull request closed without merging**